.circleci/config.yml

```diff
@@ -227,6 +227,13 @@ jobs:
 
       - run: *install-npm-modules
 
+      - run:
```

Comment:
You should be able to reuse anchors like this: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/.circleci/config.yml#L56

Feel free to create new ones and apply to android-build, if necessary, so we keep it DRY.

Reply:
Sure 👍